File: 058/cpp/main.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int cnt = 0;
        for (int i = s.size() - 1; i > -1; i--) {
            if (s[i] != ' ') {
                cnt += 1;
            } else if (cnt > 0) {
                break;
            }
        }
        return cnt;
    }
};

File: 058/rust/main.rs
impl Solution {
    pub fn length_of_last_word(s: String) -> i32 {
        let mut cnt = 0;
        let s = s.as_bytes();
        for i in (0..s.len()).rev() {
            let c = s[i] as char;
            if c != ' ' {
                cnt += 1;
            } else if cnt > 0 {
                break;
            }
        }
        return cnt;
    }
}
